packages/web/src/components/pages/TestSuites/TestSuitesList/TestSuitesList.tsx

```diff
@@ -82,7 +84,7 @@ const TestSuitesList: FC = () => {
       queryFilters={filters}
       setQueryFilters={setFilters}
       data={testSuites}
-      isLoading={isLoading}
+      isLoading={isLoading || !isSystemAvailable}
```

Comment:
Should it be marked as loading when the system is not available?

Reply:
adjusted the logic [similarly to the tests ](https://github.com/kubeshop/testkube-dashboard/pull/896/files/94ab43bfb5dc97e8e8e805cfe84b22d198398132#r1350120235)